pkgs/build-support/node/fetch-npm-deps/src/main.rs

```diff
@@ -44,6 +46,10 @@ fn to_new_packages(
     let mut new = HashMap::new();
 
     for (name, mut package) in old_packages {
+        if package.bundled {
+            continue;
+        }
```

Comment:
Could you add some way to understand why packages should be skipped when bundled, such as via a self-documenting helper method or a comment?